string object_name
float64 x
float64 y
float64 theta
string table_description
